src/client/Microsoft.Identity.Client/Platforms/Features/OpenTelemetry/OtelInstrumentation.cs

```diff
@@ -162,6 +168,8 @@ public void IncrementSuccessCounter(string platform,
                         new(TelemetryConstants.MsalVersion, MsalIdHelper.GetMsalVersion()),
                         new(TelemetryConstants.Platform, platform),
                         new(TelemetryConstants.ApiId, apiId),
+                        new(TelemetryConstants.CallerSdkId, callerSdkId ?? ""),
+                        new(TelemetryConstants.CallerSdkVersion, callerSdkVersion ?? ""),
```

Comment:
```suggestion
                        new(TelemetryConstants.CallerSdkVersion, callerSdkVersion ?? string.Empty),
```

Consider using something that will be more readable when reviewing though as well e.g. "Unknown"

Is this some value a caller optionally sets up when building out the CCA/PCA? Is there an update to our internal libraries which leverage this?